pizza/script.js

```diff
@@ -1,25 +1,61 @@
 
 console.log('hello')
 
+// ------------------------------------------------
+// так работает
+// let degrees = 0;
+
+
+// function rotatePizza(speed) {
+//   let elem = document.getElementById('bigPizza');
+//   elem.style.transform = `rotate(${degrees}deg)`;
+//   console.log(77)
+//   degrees++
+// setTimeout(rotatePizza, 25);
+// console.log(speed)
+// if (degrees > 359) {
+//   degrees = 1;
+// console.log(88)
+// }
+// }
+// rotatePizza();
+
+// ------------------------
+// хочу сделать функцию, чтоб в неё передадь любую картинку и скорость и запускать по клику например
+// но не работает
+// let degrees = 0;
+// const rotateImg =  document.getElementById('bigPizza');
+
+// function rotatePizza(elem, speed) {
+//   elem.style.transform = `rotate(${degrees}deg)`;
+//   console.log(77)
+//   degrees++
+// setTimeout(rotatePizza, speed);
```

Comment:
you need also pass 'elem' and 'speeg', setTimeout(() => rotatePizza(elem, speed), speed); - try it